composer.json

```diff
@@ -19,7 +19,7 @@
 	}
   ],
   "require": {
-    "php": ">=7.0,<7.4",
+    "php": ">=7.0 || <=7.4",
```

Comment:
I had to change it back to `,`.
`,` is converted into `and` and `|` is converted into `or`. So the comma was wrong in this place.
The pipe is used if you have more than one version. Like PHP 7.x and 8.x.
https://getcomposer.org/doc/articles/versions.md#version-range

<=7.4 didn't cover the current PHP version 7.4.8 because this would be **higher** than 7.4

Reply:
Ahhh, so, I learned something new today ;-) And good I was asking why you did changed that...